src/helpers/PollHandler.php

```diff
@@ -50,7 +50,7 @@ public static function createFromRequest($request)
     public static function modify(Poll $poll, $data)
     {
         if (array_key_exists('count_check', $data)) {
-            if ($data['count_check'] < $poll->options()->count()) {
+            if ($data['count_check'] <= $poll->options()->count()) {
```

Comment:
Why? could you explaine?

Reply:
Two allow the voter to choose all options allowed
for ex. if you changed count_check to 2 and you the question have 2 options, it won't modify it.
That means that you will allow the voter to choose all options -1